package.json

```diff
@@ -14,7 +14,7 @@
     "format": "prettier --write \"**/*.ts\" -l \"warn\" && tslint -p . --fix",
     "shipit": "tsc && np",
     "test": "jest && yarn test:extras",
-    "test:ci": "jest --ci && yarn test:extras",
+    "test:ci": "jest --ci --runInBand && yarn test:extras",
```

Comment:
@mattathompson - this was the issue with testing on travis CI!   I thought `--ci` would do it, but nope!

Reply:
@GantMan: Why does not running the specs serially cause issues on Travis? Might be a dumb question but, I'll take that risk

Reply:
Not a dumb question.  So tests are designed to be forked processes for speed, but CI stops forked processes in order to protect from people getting insane processing power on those public machines or causing fork bombs and killing the server.  

Result:  When Jest forks a process, the server kills it ASAP, causing closed connections before a test finishes running, resulting in an error.

Reply:
**TLDR:** The Jest process farm fights the Travis process killer and Jest loses 80% of the time causing non-deterministic testing.

Reply:
Oh, wow. Thats insane. Thanks for the explanation.